algorithms/CSharp/src/Maths/NaisMithsRule.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+
+public class NaismithsRule
+{
+    public static void printInfo(string locationStart, string locationEnd, int pathLenght, int upHill, double timeCalculation)
```

Comment:
The method name should on `CamelCase`
```suggestion
    public static void PrintInfo(string locationStart, string locationEnd, int pathLenght, int upHill, double timeCalculation)
```